features/suzie_unsubscribes_from_petition_updates.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Unsubscribing from petiton updates as Suzie
```

Comment:
Would appreciate some feedback on this cucumber test... I tested only the success case (see the commit message on the commit this test was introduced).